app/services/winning_vendor_update_auction.rb

```diff
@@ -11,10 +11,12 @@ def perform
     if user_is_winning_bidder? && status.present?
       auction.update(delivery_status: :pending_acceptance)
     elsif user_is_winning_bidder? && delivery_url.present?
-      auction.update(
+      update = auction.update(
         delivery_url: delivery_url,
         delivery_status: :work_in_progress
       )
+      AdminMailer.vendor_started_work(auction: auction).deliver_later
+      update
```

Comment:
good point -- what about putting the update below the mailer, then. since we are sending the email whether the update succeeds or not, seems like it would be OK to send the email then update the auction